drivers/ethernet/eth_stm32_hal.c

```diff
@@ -52,7 +52,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define ETH_MEDIA_INTERFACE_MII		HAL_ETH_MII_MODE
 #define ETH_MEDIA_INTERFACE_RMII	HAL_ETH_RMII_MODE
 
-
+#define ETH_TXBUF_DEF_NB (1U)
```

Comment:
Using TAB would be better here. You also don't need the brackets as the expression is simple one.